src/i18n/resources/ru/common.js

```diff
@@ -51,7 +51,7 @@ module.exports = {
   submittedForApproval: 'Направлено на утверждение',
   submittedToReview: 'Направлено на рассмотрение',
   tier: 'Уровень',
-  tiers: { high: 'Высокий', low: 'Низкий', notSelected: '' },
+  tiers: { high: 'Высокий', low: 'Низкий', medium: 'низкий', notSelected: '' },
```

Comment:
I am waiting for confirmation that these texts are correct since the current translation for 'low' seems to be the same as the new translation for 'medium'.
After looking up a bit online, there might be a mistake here in the contexts of the tiers. So, better way for confirmation than going through another PR.